pkg/logger/logger_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logger
+
+import (
+	"testing"
+)
+
+func TestLogError(t *testing.T) {
+	err := struct {
+		Error string
+	}{
+		"Test",
+	}
+	LogError(err)
+}
+func TestLogInfo(t *testing.T) {
```

Comment:
add a blank line between functions? Are we using any Go linter for the repo to check some basic formatting issues?